{"name": "test_sample2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 6 == 5", "trace": "def test_sample2():  # Testcase 2\n>       assert 6 == 5\nE       assert 6 == 5\n\nDEC\\4_DEC_2023\\test_labapi005.py:7: AssertionError"}, "start": 1705212173638, "stop": 1705212173638, "uuid": "481f4dc8-9695-41be-8e56-06eef8996158", "historyId": "7e91c52565355de3d1dc2d283cc1eac4", "testCaseId": "7e91c52565355de3d1dc2d283cc1eac4", "fullName": "DEC.4_DEC_2023.test_labapi005#test_sample2", "labels": [{"name": "parentSuite", "value": "DEC.4_DEC_2023"}, {"name": "suite", "value": "test_labapi005"}, {"name": "host", "value": "REYA"}, {"name": "thread", "value": "13824-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "DEC.4_DEC_2023.test_labapi005"}]}